db/sqlite_manager.py

```
@@ -24,13 +25,14 @@ class SQLiteManager:
     def __init__(self):
         """Initialize the SQLite manager."""
         self._ensure_dir_exists()
+        self.logger = logging.getLogger(__name__)
 
     def _ensure_dir_exists(self):
         """Ensure the database directory exists."""
         try:
             os.makedirs(os.path.dirname(SQLITE_PATH), exist_ok=True)
         except Exception as e:
```

Ruff (0.14.6)

Do not catch blind exception: `Exception`

(BLE001)